### ISSUE TITLE
[Tarea 1] Cargar CSV 

### ISSUE THREAD

**Issue by KnowYourselves** [IIC2523/2020-2-Tarea1#4]
Hola!

¿Nuestro programa puede cargar la totalidad del CSV, o debemos leerlo de a poco?

Gracias!

**Comment by FranLucchini** [comment]
Sí, pueden cargar la totalidad del archivo. Pueden hacerlo como prefieran o les quede más cómodo.
